jukebox_radio/users/views/get_settings_view.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.auth.mixins import LoginRequiredMixin
+
+from jukebox_radio.core.base_view import BaseView
+from jukebox_radio.core.utils import generate_spotify_authorization_uri
+
+User = get_user_model()
+
+
+class UserGetSettingsView(BaseView, LoginRequiredMixin):
+    def get(self, request, **kwargs):
+        """
+        Get the settings of a user.
+        """
+        return self.http_response_200({
```

Comment:
Nice and easy endpoint 😄

Note: `self.http_response_200` inherits from `BaseView`. I made `BaseView`, which is just a small wrapper around the Django view with some extra goodies (like `http_response_200`)